src/imprint_solids.cpp

```diff
@@ -84,19 +86,26 @@ imprint(document &doc)
 int
 main(int argc, char **argv)
 {
-	configure_spdlog();
+	configure_aixlog();
 
 	std::string path_in, path_out;
 
 	{
-		CLI::App app{"Imprint touching and overlapping solids, write to BREP file."};
-		app.add_option("input", path_in, "Path of the input file")
-			->required()
-			->option_text("input.brep");
-		app.add_option("output", path_out, "Path of the output file")
-			->required()
-			->option_text("output.brep");
-		CLI11_PARSE(app, argc, argv);
+		const char *doc = (
```

Comment:
Does it really have to be char*?

Reply:
what else should it be?  `static const char[]`?

Reply:
i was thinking std::string

Reply:
but `cxx_argp::parse` takes a `const char*` as a parameter, so that just seems to be making unnecessary work for the compiler to undo

Reply:
fair enough, I try to avoid char* as much as possible, but thats potentially a religious position :)